app/controllers/application_controller.rb

```diff
@@ -531,6 +531,13 @@ def filesystem_download
     send_data fs.contents, :filename => fs.name
   end
 
+  def adv_search_text_clear
+    if @_params[:controller] == "vm_infra"
```

Comment:
The issue is we do not yet set @explorer value to true when we process this clear text code. Timing issue at this point. But, yes, eventually, the preference is to provide app wide fix. Need more analysis before enabling all explorer pages.